test/ode/ode_convergence_tests.jl

```diff
@@ -30,6 +30,10 @@ for i = 1:2
   @test abs(sim5.𝒪est[:l2]-3) < testTol
   sim6 = test_convergence(dts,prob,ABM32())
   @test abs(sim6.𝒪est[:l2]-3) < testTol
+  sim7 = test_convergence(dts, prob, AB4())
+  @test abs(sim7.𝒪est[:l2]-3) < testTol
```

Comment:
4?

Reply:
not getting you, could you elaborate.

Reply:
You're checking if the method is 3rd order. Shouldn't it be 4th order?

Reply:
Oh my bad, I will fix it.